code/modules/reagents/reagent_containers/patch.dm

```diff
@@ -11,7 +11,6 @@
 
 /obj/item/weapon/reagent_containers/pill/patch/New()
 	..()
```

Comment:
If New() is now just ..() then you can just remove the override altogether, as that's default behavior. This makes me wonder why @Iamgoofball had to do this override in the first place, though.